store/extension_test.go

```diff
@@ -0,0 +1,59 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerateDBExtensionActions(t *testing.T) {
+	tests := []struct {
+		oldDBExtensionMap map[extensionKey]*dbExtensionRaw
+		newDBExtensionMap map[extensionKey]*api.DBExtensionCreate
+		wantDeletes       []*api.DBExtensionDelete
+		wantCreates       []*api.DBExtensionCreate
+	}{
+		{
```

Comment:
Should also test oldDBExtensionMap is nil

Reply:
Done.